e2e/fixtures/manageCases/createCase/C100/submitAndPay/submitAndPay3Content.ts

```diff
@@ -0,0 +1,10 @@
+export enum SubmitAndPay3Content {
+  pageTitle = "Submit and pay",
```

Comment:
may be add pageTitle = "Submit and pay" in some common/base file rather than defining it in each page?

Reply:
Done!